python/lsst/ip/isr/ptcDataset.py

```diff
@@ -64,85 +65,85 @@ class PhotonTransferCurveDataset(IsrCalib):
         of the fit for ptcFitTye in ["POLYNOMIAL", "EXPAPPROXIMATION"].
-    ptcTurnoff : `float`
+    ptcTurnoff : `dict` [`str, `float`]
         Flux value (in ADU) where the variance of the PTC curve starts
         decreasing consistently.
-    covariances : `dict`, [`str`, `list`]
+    covariances : `dict`, [`str`, `np.ndarray`]
         Dictionary keyed by amp names containing a list of measured
         covariances per mean flux.
-    covariancesModel : `dict`, [`str`, `list`]
+    covariancesModel : `dict`, [`str`, `np.ndarray`]
         Dictionary keyed by amp names containinging covariances model
         (Eq. 20 of Astier+19) per mean flux.
-    covariancesSqrtWeights : `dict`, [`str`, `list`]
+    covariancesSqrtWeights : `dict`, [`str`, `np.ndarray`]
         Dictionary keyed by amp names containinging sqrt. of covariances
         weights.
-    aMatrix : `dict`, [`str`, `list`]
+    aMatrix : `dict`, [`str`, `np.ndarray`]
         Dictionary keyed by amp names containing the "a" parameters from
         the model in Eq. 20 of Astier+19.
-    bMatrix : `dict`, [`str`, `list`]
+    bMatrix : `dict`, [`str`, `np.ndarray`]
         Dictionary keyed by amp names containing the "b" parameters from
         the model in Eq. 20 of Astier+19.
-    covariancesModelNoB : `dict`, [`str`, `list`]
+    covariancesModelNoB : `dict`, [`str`, `np.ndarray`]
         Dictionary keyed by amp names containing covariances model
         (with 'b'=0 in Eq. 20 of Astier+19)
         per mean flux.
-    aMatrixNoB : `dict`, [`str`, `list`]
+    aMatrixNoB : `dict`, [`str`, `np.ndarray`]
         Dictionary keyed by amp names containing the "a" parameters from the
         model in Eq. 20 of Astier+19
         (and 'b' = 0).
-    finalVars : `dict`, [`str`, `list`]
+    finalVars : `dict`, [`str`, `np.ndarray`]
         Dictionary keyed by amp names containing the masked variance of the
         difference image of each flat
         pair. If needed, each array will be right-padded with
         np.nan to match the length of rawExpTimes.
-    finalModelVars : `dict`, [`str`, `list`]
+    finalModelVars : `dict`, [`str`, `np.ndarray`]
         Dictionary keyed by amp names containing the masked modeled
         variance of the difference image of each flat pair. If needed, each
         array will be right-padded with np.nan to match the length of
         rawExpTimes.
-    finalMeans : `dict`, [`str`, `list`]
+    finalMeans : `dict`, [`str`, `np.ndarray`]
         Dictionary keyed by amp names containing the masked average of the
         means of the exposures in each flat pair. If needed, each array
         will be right-padded with np.nan to match the length of
         rawExpTimes.
-    photoCharge : `dict`, [`str`, `list`]
+    photoCharges : `dict`, [`str`, `np.ndarray`]
          Dictionary keyed by amp names containing the integrated photocharge
          for linearity calibration.
 
```

Comment:
Blow this line is the version. Does it need to be increased after the changes in this ticket? If so, will it affect backward compatibility?